### ISSUE TITLE
ActiveRecord::Tasks::DatabaseTasks regression when used outside of Rails

### ISSUE THREAD

**Issue by ForTheYin** [rails/rails#36754]
### Steps to reproduce
Clone project from https://github.com/ForTheYin/postshift and use the `beta/rails-6-support` branch.

```
bundle install
appraisal install
appraisal ar-6.0rc2 rake spec:db:migrate
```

It appears that the [original issue](https://github.com/rails/rails/issues/35902) was fixed in https://github.com/rails/rails/pull/36023, but I think https://github.com/rails/rails/pull/36560 brought it back.

### Expected behavior
```
bundle install
appraisal install
appraisal ar-6.0rc2 rake spec:db:migrate
== 1 CreateDataTypesTable: migrating ==========================================
-- adapter_name()
   -> 0.0000s
-- adapter_name()
   -> 0.0000s
-- adapter_name()
   -> 0.0000s
-- create_table("data_types", {:id=>:integer})
   -> 0.2656s
== 1 CreateDataTypesTable: migrated (0.2659s) =================================
```

### Actual behavior
```
bundle install
appraisal install
appraisal ar-6.0rc2 rake spec:db:migrate
NameError: uninitialized constant ActiveRecord::Tasks::DatabaseTasks::Rails
~/.rvm/gems/ruby-2.6.2/gems/activerecord-6.0.0.rc2/lib/active_record/tasks/database_tasks.rb:157:in `for_each'
~/.rvm/gems/ruby-2.6.2/gems/activerecord-6.0.0.rc2/lib/active_record/railties/databases.rake:28:in `block (2 levels) in <top (required)>'
~/.rvm/gems/ruby-2.6.2/gems/activerecord-6.0.0.rc2/lib/active_record/railties/databases.rake:23:in `block in <top (required)>'
~/.rvm/gems/ruby-2.6.2/gems/activerecord-6.0.0.rc2/lib/active_record/railties/databases.rake:7:in `<top (required)>'
...
```

### System configuration
**Rails version**: 6.0.0.rc2

**Ruby version**: 2.6.2

**Comment by eileencodes** [comment]
Thanks @ForTheYin - working on a fix.

**Comment by eileencodes** [comment]
Fixed in 1d83ab936672790ea3abd407df8c16d696aaef70 on master and ff6256368baad186d5b0079d23893588bf924689 on 6-0-stable.

I didn't add a test because I really don't know how to test this.
